OneCommunityGlobal/HighestGoodNetworkApp
# Raul make user profile page responsive

# Description
(USER PROFILE COMPONENT - PRIORITY LOW) 

> 1. (PRIORITY LOW) Jae: Optimize user profiles for phones and tablets.
>     a. A lot has been added to this page. What can we do to make it work better on phones and tablets? Please submit your suggestions to Jae before putting a lot of work into this. 

## Mainly changes explained:
The user profile page was edited to be responsive. The blue square section was adjusted and the tabs were replaced by a menu list. Every item on the list opens a modal with the respective functions.

## How to test:
1. git checkout raul_make_userprofile_responsive
2. npm install
3. npm run start:local
4. log in as an admin or any other role
5. click on Welcome -> View Profile, as below:
![Screen Shot 2023-01-31 at 22 29 49](https://user-images.githubusercontent.com/29555732/215922839-4bee9c58-4f37-420c-ab47-c3fb6b33fae8.png)
6. Verify if the page is responsive and working on small screens (you can test it by changing the resolutions with the inspection resource on Chrome). The proper resolutions to test are **between 425px and 1024 px**.
7. Also, **focus on the functionalities**. **This PR is not supposed to style the page** but make it fit on phone screens.

### Here are two videos to help you:

1. User profile page behavior on the development branch (BEFORE):

https://user-images.githubusercontent.com/29555732/215923563-4a7ab796-7551-483c-85c9-4ec4e8c07216.mov

2. User profile page behavior on this branch (AFTER):

https://user-images.githubusercontent.com/29555732/215923131-b6c64241-4fbf-4963-8023-781d19b2458c.mov
